<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>slugsControl</class>
 <widget class="QWidget" name="slugsControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>494</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string>Control widget for the SLUGS autopilot.</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="5,0,0,0,0,0,0,0,0,0" columnstretch="5,0">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="4" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>180</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <property name="toolTip">
       <string>Set mid-level commands for the MAV.</string>
      </property>
      <property name="statusTip">
       <string>Set mid-level commands for the MAV.</string>
      </property>
      <attribute name="title">
       <string>Mid-Level</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Altitude (m)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="altitudeSpinBox">
           <property name="toolTip">
            <string>Altitude to maintain in meters.</string>
           </property>
           <property name="statusTip">
            <string>Altitude to maintain in meters.</string>
           </property>
           <property name="maximum">
            <double>1500.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Airspeed (m/s)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="airspeedSpinBox">
           <property name="toolTip">
            <string>Airspeed to maintain (in meters per second).</string>
           </property>
           <property name="statusTip">
            <string>Airspeed to maintain (in meters per second).</string>
           </property>
           <property name="maximum">
            <double>500.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Turn Rate (rad/s)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="turnRateSpinBox">
           <property name="toolTip">
            <string>Turn rate to hold (in radians per second).</string>
           </property>
           <property name="statusTip">
            <string>Turn rate to hold (in radians per second).</string>
           </property>
           <property name="maximum">
            <double>180.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="setMidLevelButton">
           <property name="toolTip">
            <string>Sends mid-level flight parameters to MAV.</string>
           </property>
           <property name="statusTip">
            <string>Sends mid-level flight parameters to MAV.</string>
           </property>
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="getMidLevelButton">
           <property name="toolTip">
            <string>Read mid-level flight parameters from MAV.</string>
           </property>
           <property name="statusTip">
            <string>Read mid-level flight parameters from MAV.</string>
           </property>
           <property name="text">
            <string>Get</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <property name="toolTip">
       <string>Select manual control surfaces for selective passthrough.</string>
      </property>
      <property name="statusTip">
       <string>Select manual control surfaces for selective passthrough.</string>
      </property>
      <attribute name="title">
       <string>Passthrough</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="elevatorCheckBox">
           <property name="toolTip">
            <string>Manually control the elevator.</string>
           </property>
           <property name="statusTip">
            <string>Manually control the elevator.</string>
           </property>
           <property name="text">
            <string>Elevator</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="rudderCheckBox">
           <property name="toolTip">
            <string>Manually control the rudder.</string>
           </property>
           <property name="statusTip">
            <string>Manually control the rudder.</string>
           </property>
           <property name="text">
            <string>Rudder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="throttleCheckBox">
           <property name="toolTip">
            <string>Manually control the throttle.</string>
           </property>
           <property name="statusTip">
            <string>Manually control the throttle.</string>
           </property>
           <property name="text">
            <string>Throttle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="aileronsCheckBox">
           <property name="toolTip">
            <string>Manually control the ailerons.</string>
           </property>
           <property name="statusTip">
            <string>Manually control the ailerons.</string>
           </property>
           <property name="text">
            <string>Ailerons</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="setPassthroughButton">
         <property name="toolTip">
          <string>Sets the selected control surfaces for manual control.</string>
         </property>
         <property name="statusTip">
          <string>Sets the selected control surfaces for manual control.</string>
         </property>
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Choose the MAVs navigation mode.</string>
     </property>
     <property name="statusTip">
      <string>Choose the MAVs navigation mode.</string>
     </property>
     <property name="title">
      <string>Navigation Mode</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item row="2" column="0">
         <widget class="QPushButton" name="midLevelModeButton">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Hold specified mid-level commands.</string>
          </property>
          <property name="statusTip">
           <string>Hold specified mid-level commands.</string>
          </property>
          <property name="text">
           <string>Mid-Level</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="linePatrolModeButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Line Patrol</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="waypointModeButton">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Navigate to waypoints.</string>
          </property>
          <property name="statusTip">
           <string>Navigate to waypoints.</string>
          </property>
          <property name="text">
           <string>Waypoint</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="passthroughModeButton">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Manually pilot all control surfaces.</string>
          </property>
          <property name="statusTip">
           <string>Manually pilot all control surfaces.</string>
          </property>
          <property name="text">
           <string>Passthrough</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="isrModeButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Circles an ISR point.</string>
          </property>
          <property name="statusTip">
           <string>Circles an ISR point.</string>
          </property>
          <property name="text">
           <string>ISR</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="selectivePassthroughModeButton">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Manually pilot selected control surfaces.</string>
          </property>
          <property name="statusTip">
           <string>Manually pilot selected control surfaces.</string>
          </property>
          <property name="text">
           <string>Selective Pass.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="lastActionLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Status label</string>
     </property>
     <property name="statusTip">
      <string>Status label</string>
     </property>
     <property name="text">
      <string>No actions executed so far</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="controlStatusLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>10</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Currently controlled system.</string>
     </property>
     <property name="statusTip">
      <string>Currently controlled system.</string>
     </property>
     <property name="text">
      <string>UNCONNECTED</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="armDisarmButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Arms or disarms the MAV's engines.</string>
     </property>
     <property name="statusTip">
      <string>Main control button</string>
     </property>
     <property name="text">
      <string>ARM</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
